examples/spark/spark-driver.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
All other specs in spark example are also in JSON, so should I convert all to YAML or leave as it is. I see there are few examples which have specs in YAML and whereas others in JSON. So any policy towards specs format ?